css/style.css

```diff
@@ -1,9 +1,199 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 160px;
+  font-size: 1em;
+}
+
+header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 30px 10px;
+}
+
+.logo {
+  width: 50px;
+  height: 50px;
+  margin-left: 50px;
+}
+
+.navbar li {
+  list-style: none;
+  display: inline-block;
+  padding: 0px 20px;
+}
+
+.navbar a:link {
+  text-decoration: none;
+  color: grey;
+  transition: all 0.3s ease 0s;
+}
+
+.navbar a:visited {
+  color: darkgray;
+}
+
+.navbar a:hover {
+  color: rgb(255, 60, 0);
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 600px;
+  background-image: url("/img/first-background.jpg");
+}
+h1 {
+  color: white;
+  font-weight: lighter;
+  font-size: 60px;
+}
+
+h2 {
+  color: white;
+  font-weight: lighter;
+  font-size: 45px;
+}
+
+.button {
+  background-color: rgb(250, 99, 11);
+  border: none;
+  padding: 3px 15px;
+  border-radius: 5%;
```

Comment:
Nice. Probably best to set the border-radius in pixels, unless you're going for a full circle like in the footer in which case you can use `50%`